templates/docs/examples/base/code-block.html

```diff
@@ -0,0 +1,14 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Code - inline{% endblock %}
+
+{% block standalone_css %}base{% endblock %}
+
+{% block content %}
+  <pre>
+  <code>bolla.internal
```

Comment:
As we correctly suggest `<pre><code>` should be used for code blocks, so would be good to give good example, so instead of command line output we should put some basic code here.

I guess something simple would be enough:

```
function() {
  alert("Hello world");
}
```

Reply:
updated